packages/extension-api/src/extension-api.d.ts

```diff
@@ -2920,8 +2920,7 @@ declare module '@podman-desktop/api' {
     /**
      * Store a new value for key in the context.
      * This can be used in enablement of command or with the when property.
-     * The key should consists of '<extension-
-     * id>.<actual-key>'.
+     * The key should consists of '"extension-id"."actual-key"'.
```

Comment:
hint: could have been a separated PR saying that it'll cause troubles if exporting to html